badges/8BadgesOnA3.svg

```diff
@@ -10,8 +10,8 @@
    xmlns:xlink="http://www.w3.org/1999/xlink"
    xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
    xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
-   width="297mm"
-   height="420mm"
+   width="paper_width"
```

Comment:
Is the file still editable using inkscape? Changing the size will not change the positions of the elements. Could you check?

Reply:
I checking it, will update PR if necessary  ASAP